src/dattobd.h

```diff
@@ -77,13 +77,10 @@ struct dattobd_info{
 	char bdev[PATH_MAX];
 };
 
-struct dattobd_version {
-    char version_string[20]; // null terminated string for returning version number, allow for a date.
-};
-
 struct dattobd_active_device_info {
-    int count; // in/out  in = max to supply, out = number of info structs returned
-    struct dattobd_info first; // we return an array of these back to back
+	char version_string[20]; // null terminated string for returning version number, allow for a date.
+	int count; // in/out  in = max to supply, out = number of info structs returned
+	struct dattobd_info first[0]; // we return an array of these back to back
```

Comment:
can we rename this to something more descriptive (especially since its not just the first of these anymore).